test/bond/test/james.cljc

```diff
@@ -49,6 +49,17 @@
       (is (= [4] (-> target/bar bond/calls second :args)))
       (is (= [5] (-> target/bar bond/calls last :args))))))
 
+#?(:clj
```

Comment:
I was thinking `with-stub!` should be omitted entirely in CLJS because I thought it didn't have the ability to load var metadata, but apparently that's no longer true, so yeah that's a good idea.